bci-study-alpha-waves/README.md

```diff
@@ -114,7 +114,21 @@ feeding live data to Neuropype and just using a recording file.
 
 #### Control RC tank
 
-WIP...
+To control RC tank you need to do these steps:
+
+- Turn on helmet (PC mode), input the DONGLE
+- Open BCI GUI
+- Start 16 channels
+- Select widgets - EEG, FFT, NETWORKING
+- Start LSL streams (Protocol LSL, TimeSeriesFilt)
+- Put helmet on head
+- Turn on RC tank (power and motor switch) -> HC-06 model should blink
+- Turn on bluetooth  -> connect to HC-06 (password is 1234)
+- Run `4_AlphaWavesOnlineWakefulRelaxationResult.pyp` pypeline
+- Run `bci_bl_control_alpha_study.py` -> choose correct COM port
+- Lower treshold values if you need
```

Comment:
We don't have these, please describe what to edit in the pipeline itself, mb with a screenshot

Reply:
if sample_result[0] > 0.8: ???????